cpansa/CPANSA-GitLab-API-v4.yml

```diff
@@ -0,0 +1,11 @@
+---
+- id: CPANSA-GitLab-API-v4-2023-01
```

Comment:
This needs the affected_versions key.

Reply:
added `affected_versions: ~`

Reply:
@stigtsp I'm also not a fan of it being null. Even in the JSON Schema we created we stated that `affected_versions` must always be defined.

So, if it affects *only* the current version, I think it should say `"=0.26"` and we should have some sort of mechanism to monitor future releases to see if there was a fix; or, maybe simpler, just make a blank statement of `">= 0.26"` and let the author contact CPANSA about having fixed the code so we can update the entry to either a single release or a range.

But if it affects *all* versions, I think it should say `">0"`, maybe?

Reply:
Yes, if it is still a problem, find the earliest version where it is a problem and use `>=` with no upper range.